index.html

```diff
@@ -37,7 +39,16 @@
     <div class="posts-container">
       <div id="feature-posts">
         <h1>Featured Posts</h1>
-        Should we feature some popular posts here, make it a slider?
+        <hr>
+          <div class="card-list"></div>
+          <div class="card-list"></div>
+          <div class="card-list"></div>
+          <br>
+          <div style="text-align:center">
```

Comment:
According to our coding convention, style shouldn't be inline a tag. 
Name your div as class or id, define its style in .css file.